Action Fails with `sh: 3: set: Illegal option -o pipefail`

Changes to the upstream install script have caused this action to start failing. This is noted in [the most recent PR](https://github.com/kevincobain2000/gobrew/pull/206).

```
> Run kevincobain2000/action-gobrew@e6b1116c09b28a0e20a753384c90bb7fa95ec198
> Run echo "$HOME/.gobrew/current/bin" >> $GITHUB_PATH
  echo "$HOME/.gobrew/current/bin" >> $GITHUB_PATH
  echo "$HOME/.gobrew/bin"  >> $GITHUB_PATH
  echo "$HOME/go/bin"  >> $GITHUB_PATH
  curl -sLk https://raw.githubusercontent.com/kevincobain2000/gobrew/master/git.io.sh | sh
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    ...
sh: 3: set: Illegal option -o pipefail
Error: Process completed with exit code 2.
```

--- a/action.yml
+++ b/action.yml
@@ -22,7 +22,7 @@ runs:
         echo "$HOME/.gobrew/current/bin" >> $GITHUB_PATH
         echo "$HOME/.gobrew/bin"  >> $GITHUB_PATH
         echo "$HOME/go/bin"  >> $GITHUB_PATH
-        curl -sLk https://raw.githubusercontent.com/kevincobain2000/gobrew/master/git.io.sh | sh
+        curl -sLk https://raw.githubusercontent.com/kevincobain2000/gobrew/master/git.io.sh | bash
       shell: bash
 
     - name: gobrew use

